packages/core/src/internal/execution/types/messages.ts

```diff
@@ -79,6 +82,7 @@ export interface DeploymentExecutionStateInitializeMessage {
   futureId: string;
   futureType: DeploymentExecutionStateFutureTypes;
   strategy: string;
+  strategyConfig: PassedStrategyConfig;
```

Comment:
What happens if we read an existing journal that doesn't have this property?

Reply:
I guess the reducers should handle it? Can they?

Reply:
We can choose a behaviour to deal with the absence of a `strategyConfig`. Should reconciliation fail or pass if the previous run was on the previous version.

I think we can say it returns the `{}` and either pass or fail reconciliation on that, as the previous version is by definition `basic` with no config.